utility/split-log-into-stages.sh

```diff
@@ -29,7 +29,7 @@ do
     echo "copying '$stage' log to $stage_log_filename"
     grep --perl-regexp "$this_stage_regexp" "$full_log" | gzip > "$stage_log_filename"
 
-    if grep "Failed in branch" temp.txt
+    if grep "Failed in branch" $stage_log_filename
```

Comment:
This is a `.gz` file. Please use `zgrep` or `zcat | grep`.

Reply:
TIL